visidata/loaders/http.py

```diff
@@ -1,7 +1,8 @@
 from visidata import Path, RepeatFile, options, vd
 
 content_filetypes = {
-    'tab-separated-values': 'tsv'  # thanks @lindner
+    'tab-separated-values': 'tsv',  # thanks @lindner
+    'json': 'json'
```

Comment:
I don't think this should be necessary, as it uses the subtype as the filetype if it's not present in content_filetypes (see line 25).